src/main/query_result.cpp

```diff
@@ -95,6 +96,9 @@ struct DuckDBArrowSchemaHolder {
 	vector<ArrowSchema> children = {};
 	// unused in children
 	vector<ArrowSchema *> children_ptrs = {};
+	//! used for nested structures
+	std::list<ArrowSchema> nested_children = {};
```

Comment:
Unless I'm mistaken it seems that `nested_children` and `nested_children_ptr` are not actually used, is that correct?

Reply:
It is on SetArrowFormat

Reply:
I'll remove the initializers for lists and vectors ;-)

Reply:
Ah got it, thanks